flocx_market/tests/unit/objects/test_object_bid.py

```diff
@@ -0,0 +1,67 @@
+from flocx_market.objects import bid
+import datetime
+import unittest.mock as mock
+now = datetime.datetime.utcnow()
+
+
+test_bid_1 = dict(creator_bid_id="1259a51-b4d6-497d-9f75-f56c409305c8",
+                  creator_id="12a59a51",
+                  server_quantity=2,
+                  start_time=now,
+                  end_time=now,
+                  duration=16400,
+                  status="available",
+                  server_config_query={'foo': 'bar'},
+                  cost=11.2)
+
+test_bid_2 = dict(creator_bid_id="12a59a51-b4d6-497d-9f75-f56c409305c8",
+                  creator_id="12a9a51",
+                  server_quantity=2,
+                  start_time=now,
+                  end_time=now,
+                  duration=16400,
+                  status="available",
+                  server_config_query={'foo': 'bar'},
+                  cost=11.5)
+
+
+@mock.patch('flocx_market.db.sqlalchemy.api.bid_create')
+def test_create(bid_create):
+    bid.Bid.create(test_bid_1)
+    bid_create.assert_called_once()
+
+
+@mock.patch('flocx_market.db.sqlalchemy.api.bid_create')
+@mock.patch('flocx_market.db.sqlalchemy.api.bid_get')
+def test_get(bid_create, bid_get):
+    ret = bid.Bid.create(test_bid_1)
```

Comment:
You don't need this (or to mock bid_create)

Reply:
I need the marketplace_bid_id which you get assigned when you create - not sure if that  ret = bid.Bid(**test_bid_1) will also assign marketplace_bid_id.